generators/spring-boot/generator.ts

```diff
@@ -71,12 +72,11 @@ const { CASSANDRA, COUCHBASE, MONGODB, NEO4J, SQL } = databaseTypes;
 const { MICROSERVICE, GATEWAY } = applicationTypes;
 const { KAFKA, PULSAR } = messageBrokerTypes;
 const { ELASTICSEARCH } = searchEngineTypes;
+const { NO: NO_CLIENT } = clientFrameworkTypes;
 
 const { BYTES: TYPE_BYTES, BYTE_BUFFER: TYPE_BYTE_BUFFER } = fieldTypes.RelationalOnlyDBTypes;
 const { CUCUMBER, GATLING } = testFrameworkTypes;
 export default class SpringBootGenerator extends BaseApplicationGenerator {
-  fakeKeytool;
```

Comment:
See https://github.com/jhipster/generator-jhipster/pull/27289/files#r1761364147.

Reply:
It is accessed within the angular templates: I think we have to tailor `application` and `entity` types per generator in order to introduce specific attributes/methods.
Generator should be here for orchestration (thus being stateless), the only data objects Application, entity, platform (and even maybe fields/relationships).
I'll try to add generics on generator definition and see where it goes

Reply:
I’ve tried to make generator specific application types before.
In the end it’s not worthy because many generator uses types of another generator.
Maybe new generators like java:* and javascript:* it may be possible.

Reply:
Now it works well and is pretty simple (thanks for the architecture enhancements ;-)).
I hope that detangling code by putting Master objects (Application/Field..) fields into their respective generators (or at least at the right level) will simplify the base and make it more clear (i.e. like cleaning generator/base/base-application/types.d).